platform-sdk/swirlds-platform-core/src/main/java/module-info.java

```diff
@@ -126,6 +126,7 @@
     requires com.swirlds.test.framework;
     requires com.swirlds.logging;
     requires com.swirlds.cli;
+    requires com.hedera.platform.bls.api;
```

Comment:
Where can I find that module?

Reply:
This is in the `hedera-bls-cryptography` repo (see PR body for more info)